README.md

```diff
@@ -1 +1,24 @@
-This repository is for Salt formulas.
+# SUSE Salt Formulas
+
+This repository contains Salt Formulas to be shipped for `openSUSE` and `SUSE Linux Enterprise Server`. The Formulas can be used with stand-alone Salt or within [Uyuni](https://github.com/uyuni-project/uyuni) / [SUSE Manager](https://www.suse.com/products/suse-manager/) offering UI support for configuring the states.
```

Comment:
It'd sound better, maybe more accurate instead of saying "configuring the states" -> changing pillar data or configuring the formulas?

Reply:
Thanks, changed the text slightly.